src/RoadRegistry.BackOffice/Core/RoadNodeGeometryTaken.cs

```diff
@@ -1,6 +1,6 @@
 namespace RoadRegistry.BackOffice.Core
 {
-    public class RoadNodeGeometryTaken : Warning
+    public class RoadNodeGeometryTaken : Error
```

Comment:
Waarom van Warning naar Error?

Reply:
In a previous commit I changed it from Error to Warning, but the change was meant for the `RoadNodeTooClose` class instead of this class.

Reply:
lgtm